kaldb/src/main/java/com/slack/kaldb/logstore/search/SearchQuery.java

```diff
@@ -49,7 +50,7 @@ public String toString() {
         + ", bucketCount="
         + bucketCount
         + ", chunkIds="
-        + chunkIds
+        + Strings.join(chunkIds, ',')
```

Comment:
I'd prefer if we don't use a log4j library for this given there exists one in the standard JDK

https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

Reply:
Good catch. Dang auto-complete. That's what I thought I used. Fixed now.